src/client/camera.cpp

```diff
@@ -467,21 +467,62 @@ void Camera::update(LocalPlayer* player, f32 frametime, f32 busytime, f32 tool_r
 	 * Otherwise, default to m_cache_fov
 	 */
 
-	f32 fov_degrees;
-	PlayerFovSpec fov_spec = player->getFov();
-	if (fov_spec.fov > 0.0f) {
+	static f32 fov_degrees;
+	static bool fov_transition_active = false;
+
+	// Smooth transition variables
+	static f32 old_fov, fov_diff, transition_time;
```

Comment:
static thread_local please, else it will be stored across game sessions

Reply:
Alright